data/0rigin-c0de.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "Sunil Kumar",
+	"type": "personal",
+	"bio": "I'm a self-taught from **India**",
```

Comment:
@0rigin-c0de you might want to specify exactly what you are self-taught at, for example "I'm a self taught `developer` from India'

However as this is not a blocker I will merge your Pull Request.  If you would like to many any changes please raise a new Pull Request (and you may want to have a look at our Editing Guide in Docs too: https://linkfree.eddiehub.io/docs/how-to-guides/editing)